app/templates/components/password/forgot-password.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <form>
+
+    {{input name="email" autocapitalize="off" type="text" placeholder='Your email' value=email key-down="keyDown"}}
+
+    <div class="input-group">
+      <input class="button default {{if hasError "has-error"}}" name="forgot-password" type="submit" value='Forgot Password' {{action forgotPassword email}} />
```

Comment:
I'm asking which one would have it. I think the button having it is a little weird, but the text having it makes more sense.